scout/server/blueprints/phenotypes/views.py

```diff
@@ -14,3 +15,18 @@ def hpo_terms():
     """Render search box and view for HPO phenotype terms"""
     data = controllers.hpo_terms(store=store)
     return data
+
+
+@hpo_bp.route("/api/v1/phenotypes", methods=["GET"])
+@public_endpoint
+def api_hpo_terms():
+    """Public API for HPO phenotype terms: retrieve all HPO terms
+
+    Args:
+        store(adapter.MongoAdapter):  an adapter to the scout database
```

Comment:
Did you copy this from some other function? Store is not passed to this function

Reply:
Certainly copied, but for a previous iteration. That part moved to the controller!